src/components/v5/shared/RichText/RichText.tsx

```diff
@@ -119,7 +120,9 @@ const RichText: FC<RichTextProps> = ({
                 className={clsx('transition sm:hover:text-blue-400 text-left', {
                   'text-gray-900': characterCount,
                   'text-gray-400': !characterCount,
+                  'sm:hover:text-blue-400': !isDisabled,
```

Comment:
```suggestion
                className={clsx('transition text-left', {
                  'text-gray-900': characterCount,
                  'text-gray-400': !characterCount,
                  'sm:hover:text-blue-400': !isDisabled,
```

sm:hover:text-blue-400 should no longer be applied by default as it is now applied only when !isDisabled.